formatTest/typeCheckedTests/expected_output/comments.re

```diff
@@ -21,38 +34,38 @@
 /* comment ***/
 /* comment ****/
 /* comment *****/
+
```

Comment:
Yeah, this looks better more correct (I thought were were already doing this but apparently not).